studio/components/layouts/AuthLayout/AuthLayout.tsx

```diff
@@ -15,16 +15,15 @@ interface Props {
 
 const AuthLayout: FC<Props> = ({ title, children }) => {
   const { ui, meta } = useStore()
-  const { isLoading, error } = meta.tables
+  const { isInitialized, isLoading, error } = meta.tables
   const projectRef = ui.selectedProject?.ref ?? 'default'
 
   const router = useRouter()
   const page = router.pathname.split('/')[4]
 
-  const [loaded, setLoaded] = useState<boolean>(false)
+  const [loaded, setLoaded] = useState<boolean>(isInitialized)
```

Comment:
this shouldn't be a `useState`, leaves room for possibility for local state going out of sync with store.

`loaded` should be a computed value based on `isLoading` and `isInitialized`, no?

Reply:
also, if say we're watching the mobx values to trigger re-renders, a more appropriate function would be `useMemo`. 

If the mobx value changes will trigger re-renders (not too sure abt this), can just do a render computation.

Reply:
good observation actually yeah agreed - plus this "loaded" thing is common across some pages, probably requires a small refactor on the store side

i'll do this in another PR - will leave a comment first